alica-plan-designer-fx-codegeneration/src/main/java/de/unikassel/vs/alica/generator/cpp/XtendTemplates.xtend

```diff
@@ -986,6 +986,9 @@ namespace alica
                 «ENDIF»
                 /*PROTECTED REGION END*/
             protected:
+                «IF (plan.hasDynamicEntryPoint())»
+                virtual const std::unordered_map<int64_t, std::unordered_set<int64_t>>& getApplicationEntrypointContext(std::vector<int64_t> entryPointIds) override;
```

Comment:
as suggested by @abhi1287, let's use the signature:
`virtual bool getApplicationEntrypointContext(std::unordered_map<int64_t, std::unordered_set<int64_t>& entryPointMap) override;` instead to avoid dynamic memory allocations on each tick of the engine